### ISSUE TITLE
VisibleDeprecationWarning happend after I use pg 

### ISSUE THREAD

**Issue by osDanielLee** [pyqtgraph/pyqtgraph#306]
My code is really simple, and I don't know why this warning happend after I use pg.

    def drawCurve(self):
        while False == self.drawCurveLayout.isEmpty():
             it = self.drawCurveLayout.itemAt(0)
             self.drawCurveLayout.removeItem(it)
        axis = DateAxis(orientation='bottom')
        vb = CustomViewBox()
        #pw = pg.PlotWidget(viewBox=vb, axisItems={'bottom': axis}, enableMenu=False, title="plot figure")
        pw = pg.PlotWidget(enableMenu=False, axisItems={'bottom': axis}, title="plot figure")
        dataArray = [time.mktime(time.strptime(d, "%Y/%m/%d %H:%M:%S")) for d in self.dt]
        dates = np.array(dataArray,dtype=np.int32)
        '''
        Symbol to use for drawing points OR list of symbols, one per point. Default is no symbol. Options are o, s, t, d, +, or any QPainterPath
        '''
        if QtCore.Qt.Checked == self.co2CheckBox.checkState():
            pw.plot(x=dates, y=np.array(self.co2,dtype=np.int16), pen=(255,0,0), symbol='t')  #red
        if QtCore.Qt.Checked == self.tempCheckBox.checkState():
            pw.plot(x=dates, y=np.array(self.temp,dtype=np.float16), pen=(0,255,0), symbol='+')  #blue
        if QtCore.Qt.Checked == self.humiCheckBox.checkState():
            pw.plot(x=dates, y=np.array(self.humi,dtype=np.float16), pen=(0,0,255), symbol='d') #green
        self.drawCurveLayout.addWidget(pw)

the warning just looks like below:
`E:\workspace\UpSite\dist\Main_UpSite.exe\pyqtgraph\graphicsItems\ScatterPlotItem.py:186: VisibleDeprecationWarning: using a non-integer number instead of an integer will result in an error in the future`

by the way, I use python3.4,PyQt4 and pyqtgraph 0.9.

**Comment by goetzc** [comment]
I also see this, it seems to be a Numpy 1.8.0 deprecation warning, which PyQtGraph uses.

PyQtGraph 0.9.10
Numpy 1.11.0
Python 3.5.1
PyQt 5.5.1

```
/usr/lib/python3.5/site-packages/pyqtgraph/graphicsItems/ScatterPlotItem.py:186: VisibleDeprecationWarning: using a non-integer number instead of an integer will result in an error in the future
  self.atlasData[x:x+w, y:y+h] = rendered[key]
```

**Comment by campagnola** [comment]
I believe this is already fixed in the latest; will be released with 0.9.11.

**Comment by goetzc** [comment]
That is great!  When is the time frame for 0.9.11?
